includes/Handlers/Connection.php

```diff
@@ -982,6 +982,19 @@ public function is_connected() {
 	}
 
 
+	/**
+	 * Determines whether the Commerce has a manager ID.
+	 *
+	 * @since 2.1.0-dev.1
+	 *
+	 * @return bool
+	 */
+	public function has_commerce_manager_id() {
```

Comment:
Are you sure we need this helper method? Is it going to be used anywhere else? In this case I think having the whole if inside the `is_connected()` may be preferable...

Reply:
I added that to improve readability, but yeah, I will do that in the `is_connected()` method.